olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -9,3 +9,17 @@ entries:
   - name: "serverless-operator.v1.29.0"
     replaces: "serverless-operator.v1.28.0"
     skipRange: ">=1.28.0 <1.29.0"
+---
```

Comment:
Do we want to add a [package](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#olmpackage) for the catalog.

```
{
  "schema": "olm.package",
  "name": "serverless-operator",
  "defaultChannel": "latest"
}
```

Reply:
Not so sure, I guess we don't need it (Not sure what it will change on the catalog page), 
I believe we never used olm.package In the past.